specs/waku/waku.md

```diff
@@ -509,6 +524,15 @@ Known static nodes MAY also be used.
 
 ## Changelog
 
+### Version 0.4
+
+Released [February 20, 2020](https://github.com/vacp2p/specs/commit/47d81ce179282c245de8a60e977d0a8419573bb2).
+
+- Add Status Update code `0x02`
```

Comment:
```suggestion
- Add Status Update code `0x02`
```

Leftover line, it is mentioned twice

Reply:
mmh, not sure, I see it only once and I can add to the batch "This suggestion is invalid because no changes were made to the code"

Reply:
ok maybe github bug